test/integration/users.test.js

```diff
@@ -585,6 +575,60 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users/?id", function () {
+    let testUser;
+    beforeEach(async function () {
+      await addOrUpdate(userData[0], userId);
+      const { user } = await users.fetchUser({ userId });
```

Comment:
why are we fetching the user in before each? isn't this specific to a single test or is it required everywhere in this describe?